### PR TITLE
First pass on pow faucet for testnet

### PR DESCRIPTION
**Pull request by prestonvanloon**
This is the backend for the testnet site to provide test users with a small amount of GoETH.

The design is to use the faucet proto defined here https://github.com/prestonvanloon/prysm-testnet-site/blob/dev/src/proto/faucet.proto to make grpc requests from the UI. Who knew it was so easy to use grpc-web now that it has launched the GA build. 🙌 

There is also some extra yaml for kubernetes in here. This endpoint is "live" at http://api.prylabs.net, but always returns a fake fulfillment (actually checks the captcha though!)

Next steps for this faucet service:
- Broadcast a funding transaction & block the response until this request is fulfilled.

### CODE REVIEW

File: k8s/geth/faucet.deploy.yaml [hunk 1]
```diff
@@ -1,103 +1,34 @@
...
-            secretName: geth-faucet-accounts-secret
-            items: 
-              - key: json
-                path: account.json
-        - name: faucet-data
-          emptyDir: {}
+      - name: faucet-grpc
+        image: gcr.io/prysmaticlabs/prysm/faucet:latest
+        args: 
+        - --recaptcha_secret=6LdX85EUAAAAAO-_qMEOdecIzxV3eH222ekzi1Y9
```
Comment (prestonvanloon) [inline k8s/geth/faucet.deploy.yaml]: This is a test/localhost secret. No need to be alarmed, but I shouldn't have committed it. I will remove it and rotate this key, later this week.

**Comment by prestonvanloon** [comment]
Lack of generated pb.go files breaks the metalinter. I’ll move/generate those into the proto package.

File: k8s/geth/faucet.deploy.yaml (hunk 1 above)
Comment (nisdas) [reply k8s/geth/faucet.deploy.yaml]: We shouldn't merge this into master then,no ?

Comment (prestonvanloon) [reply k8s/geth/faucet.deploy.yaml]: It's not a big deal, helpful for localhost testing

**Pull request merged**